index.html

```diff
@@ -801,6 +872,56 @@ <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.6.
         <div class="container">
 
 
+<!-- legacy releases v4.0.20 -->
+<!-- 
+    <div class="break"></div> 
+    insert this code when a new update comes out to separate a new version from the current.
+-->
+<div class="smaller-margin" id="lr-12">
+    <h2 class="text-middle">Legacy releases</h2>
+    <!-- h2 tag needs to be moved to the newest version -->
+    <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.0.19...v4.0.20">v4.0.20</a></h4>
+</div>
+
+<div class="dimmed-text"><p>Release date: 2018-09-28</p></div>
+
+<ul>
+    <li>Fixed/added:</li>
+    <ol>
+        <li>Updated sortbootorder to v4.6.11</li>
+    </ol>
+    <li>Known issues:</li>
+    <ol>
+        <li>pfSense 2.4.x fails to boot when no USB stick is plugged</li>
+        <li>certain USB 3.x sticks happen to not appear in boot menu</li>
+    </ol>
+</ul>
+
+<div class="source-code-links-binaries">
+    <div class="source-code-column">
+        <div class="smaller-margin"><p>Binaries:</p></div>
+        <ul>
+            <li><a href="http://pcengines.ch/file/apu2_v4.0.20.rom.tar.gz">apu2 v4.0.20</a></li>
+            <li><a href="http://pcengines.ch/file/apu3_v4.0.20.rom.tar.gz">apu3 v4.0.20</a></li>
+            <li><a href="http://pcengines.ch/file/apu4_v4.0.20.rom.tar.gz">apu4 v4.0.20</a></li>
+            <li><a href="http://pcengines.ch/file/apu5_v4.0.20.rom.tar.gz">apu5 v4.0.20</a></li>
+        </ul>
+    </div>
+    <div class="source-code-column">
+        <div class="smaller-margin"><p>Source code:</p></div>
+        <ul>
+            <li><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.0.19...v4.0.20">coreboot v4.0.20</a></li>
+            <li><a target="_blank" href="https://github.com/pcengines/seabios/compare/rel-1.11.0.4...rel-1.11.0.5">SeaBIOS rel-1.11.0.5</a></li>
+            <li><a target="_blank" href="https://github.com/pcengines/sortbootorder/compare/v4.6.5...v4.6.9">sortbootorder v4.6.9</a></li>
```

Comment:
@Moviss the link to sortbootorder compare should be
`https://github.com/pcengines/sortbootorder/compare/v4.6.9...v4.6.11`

Reply:
@miczyg1 a tekst powinie być analogicznie: `sortbootorder v4.6.11`?